src/backend/embedding/chunkfile_Weaviateprocess/chunkfile_uploadWeaviate.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import json
+from weaviate import Client
+from sentence_transformers import SentenceTransformer
+from src.backend.weaviate.weaviate_manager import WeaviateManager
+
+# Configure logging
+logging.basicConfig(level=logging.DEBUG, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+class WeaviateSearch:
```

Comment:
We can delete this file chunkfile_uploadWeaviate.py and add all the functions to weaviate manager file